pyqtgraph/graphicsItems/TargetItem.py

```diff
@@ -1,132 +1,352 @@
+from math import atan2, pi
+
 from ..Qt import QtGui, QtCore
-import numpy as np
 from ..Point import Point
 from .. import functions as fn
 from .GraphicsObject import GraphicsObject
+from .UIGraphicsItem import UIGraphicsItem
 from .TextItem import TextItem
+from .ScatterPlotItem import Symbols
 
-
-class TargetItem(GraphicsObject):
+class TargetItem(UIGraphicsItem):
     """Draws a draggable target symbol (circle plus crosshair).
 
     The size of TargetItem will remain fixed on screen even as the view is zoomed.
     Includes an optional text label.
     """
-    sigDragged = QtCore.Signal(object)
 
-    def __init__(self, movable=True, radii=(5, 10, 10), pen=(255, 255, 0), brush=(0, 0, 255, 100)):
-        GraphicsObject.__init__(self)
+    sigPositionChanged = QtCore.Signal(object)
+    sigPositionChangeFinished = QtCore.Signal(object)
+
+    def __init__(
+        self,
+        pos=None,
+        diameter=10,
+        movable=True,
+        pen=None,
+        hoverPen=None,
+        brush=None,
+        hoverBrush=None,
+        symbol="crosshair",
+        label=None,
+        labelOpts=None,
+    ):
+        """
+        Parameters
+        ----------
+        pos : list, tuple, QPointF, or QPoint
+            Initial position of the cursor.
+        diameter : int
+            Size of the cursor in pixels.  Default is 10.
+        cursor : str
+            String that defines the shape of the cursor (can take the following
+            values for the moment : 's' (square), 'c' (circle))
```

Comment:
Hasn't the 'cursor' option been replaced by 'symbol'?

Reply:
you're absolutely right; all instances in docs or variable names have been updated 👍